astrocut/cube_cut.py

```diff
@@ -815,16 +815,23 @@ def cube_cut(self, cube_file, coordinates, cutout_size,
 
         warnings.filterwarnings("ignore", category=wcs.FITSFixedWarning)
         fits_options: Dict[str, Any] = {"mode": "denywrite", "lazy_load_hdus": True}
+
         if cube_file.startswith("s3://"):
             fits_options["use_fsspec"] = True
-            # block size should be:
-            # block_size <= m * hdul[1].section.shape[2] * hdul[1].section.shape[3] * 4 bytes
             fits_options["fsspec_kwargs"] = {"default_block_size": 10_000, "anon": True}
             # only use .section for remote data
             cube_data_prop = "section"
         else:
             fits_options["memmap"] = True
             cube_data_prop = "data"
+
+        # block size should be:
+        # block_size <= m * hdul[1].section.shape[2] * hdul[1].section.shape[3] * 4 bytes
+        if cube_file.startswith("s3://"):
```

Comment:
can we move this section inside the conditional on line 819 and it doesn't need to be in it's own location?